easee/charger.py

```diff
@@ -165,16 +190,26 @@ async def update_firmware(self):
         """Update charger firmware"""
         return await self.easee.post(f"/api/chargers/{self.id}/commands/update_firmware")
 
-    async def set_dynamic_current(self, currentP1: int, currentP2: int = None, currentP3: int = None):
+    async def set_dynamic_charger_circuit_current(self, currentP1: int, currentP2: int = None, currentP3: int = None):
         """ Set circuit dynamic current for charger """
         if self.circuit is not None:
             return await self.circuit.set_dynamic_current(currentP1, currentP2, currentP3)
         else:
             _LOGGER.info("Circuit info must be initialized for dynamic current to be set")
```

Comment:
Is this when someone has retrieved it from get_chargers only?

Reply:
It is only a name change that was necessary since I discovered and implemented the other current limits which are for chargers and not circuits. It may be that we can remove the _circuit_ methods from charger now since we have site and circuit as objects and can access them directly from HA. I'll check this later, but for now this method is merely setting the circuit dynamic current for the circuit belonging to the specific charger (so we don't need to bother about circuit ids).